components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -253,7 +253,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
     this.downloadOptsVisible = false;
 
     const reportQuery = this.reportQuery.getReportQuery();
-    const filename = `${moment(reportQuery.endDate).format('YYYY-M-D')}.${format}`;
+    const filename = `${reportQuery.endDate.format('YYYY-M-D')}.${format}`;
```

Comment:
Suggest constant width for easy filename sorting at the command line:
```suggestion
    const filename = `${reportQuery.endDate.format('YYYY-MM-DD')}.${format}`;
```

Reply:
I don't feel comfortable making this change with this PR.